macros/plugins/sqlserver/create_external_schema.sql

```diff
@@ -0,0 +1,13 @@
+{% macro sqlserver__create_external_schema(source_node) %}
```

Comment:
@guillesd Sorry for the late response. It looks to me that this certainly is the right ddl for sqlserver but there's a link to the the spark docs here? I think we should link to the sqlserver article/docs.

At the same time, looks like there's also a missing `spark__create_external_schema` with the right ddl `create database ...` which is what you linked to here + also pointed out in #100 :)

Reply:
Hi @jeremyyeo, I should've seen this sooner, my bad!

I'll fix your first point on SQL server docs

For the second point, `create schema` and `create database` are the same thing in Spark --> https://spark.apache.org/docs/latest/sql-ref-syntax-ddl-create-database.html#syntax